Documentation/operations/upgrade-warning.rst

```diff
@@ -7,8 +7,12 @@
 .. warning::
 
    Read the full upgrade guide to understand all the necessary steps before
-   performing them.
+   performing them. Pay particular attention to :ref:`current_release_required_changes`
+   before proceeding.
 
-   Do not upgrade to \ |NEXT_RELEASE|.0 before reading the section
-   :ref:`current_release_required_changes` and completing the required steps.
-   Skipping this step may lead to an non-functional upgrade.
```

Comment:
IMO I think the wording in the paragraph you removed is helpful to users because it gives a very clear warning with instructions. Users may interpret what it means to "pay particular attention" differently, but it's harder to misinterpret "do not upgrade before reading". Would you mind reverting this change? What do you think?